src/dark-mode-toggle.mjs

```diff
@@ -274,14 +272,14 @@ export class DarkModeToggle extends HTMLElement {
     this._darkCSS = null;
     this._lightCSS = null;
 
-    doc.addEventListener(COLOR_SCHEME_CHANGE, (e) => {
-      this.mode = e.detail.colorScheme;
+    doc.addEventListener(COLOR_SCHEME_CHANGE, (event) => {
+      this.mode = event.detail.colorScheme;
```

Comment:
This change doesn't help with file size -- just scratching an itch here :)

Reply:
Agreed :-)